static/styles/settings.css

```diff
@@ -167,9 +167,13 @@ h3,
     cursor: not-allowed;
 }
 
+#user_deactivate_account_button {
+    margin-right: 12px;
+}
```

Comment:
This would be better done as `margin-left` on the second element if present. It's probably not important but we don't need right margin here if the second button doesn't exist, and we should say what we mean with CSS.